wgpu-hal/src/dx12/adapter.rs

```diff
@@ -292,6 +297,13 @@ impl super::Adapter {
             bgra8unorm_storage_supported,
         );
 
+        features.set(
+            wgt::Features::SUBGROUP_COMPUTE
+                | wgt::Features::SUBGROUP_FRAGMENT
+                | wgt::Features::SUBGROUP_VERTEX,
```

Comment:
correct me, but I don't think shader model 6 supports vertex subgroup voting, see https://learn.microsoft.com/en-us/windows/win32/direct3dhlsl/hlsl-shader-model-6-0-features-for-direct3d-12#shading-language-intrinsics

Reply:
good catch, I've fixed this :)